src/game.js

```diff
@@ -0,0 +1,48 @@
+var Game = function(initialState){
+  this.initialState;
+  this.cells; 
+  this.board;
+  this.numberOfIterations;
+  this.timeout;
+  this.numberOfRows;
+  this.numberOfColumns;
+
+  this.init(initialState);
+};
+
+Game.prototype.init = function(initialState){
+  this.initialState = initialState || '-----,' +
+                                      '--x--,' +
+                                      '--x--,' +
+                                      '--x--,' +
+                                      '-----,' ;
+
+  this.cells = inputParser.parse( this.initialState ); 
+  this.board = new Board( this.cells );
+  this.numberOfIterations = 100;
+  this.timeout = 500;
+  this.numberOfRows = (this.initialState.match(/,/g) || []).length;
+  this.numberOfColumns = this.initialState.split(',')[0].split('').length;
+};
+
+Game.prototype.iterate = function(){
+  var self = this;
+  var cellsToChange =  $.map( this.board.coordinates, function(cell){
```

Comment:
@dougflip @seejee I also keep a list of the cells created from the input parser, i think i can use that instead of iterating over the array. let me ponder on this! :heart: